platform/admin/core/admin-core-api/pom.xml

```diff
@@ -73,7 +73,6 @@
         <dependency>
             <groupId>org.apache.felix</groupId>
             <artifactId>org.apache.felix.utils</artifactId>
-            <version>1.2.0</version>
```

Comment:
I don't see a version defined anywhere in the hierarchy, should it be? If I run a `mvn help:effective-pom` I get version 1.8.2, is that the desired version?